mobile-app/lib/features/components/transaction_details_action_sheet.dart

```diff
@@ -95,6 +102,41 @@ class _TransactionDetailsActionSheetState
     return 'received from';
   }
 
+  void _retryFailedTx() async {
```

Comment:
I guess this is referring to a transaction that failed to schedule... ?! in case of reversible that is...

Reply:
I am saying we don't get information if we manage to schedule a transaction but it then fails when it's trying to execute.